kevm_pyk/src/kevm_pyk/solc_to_k.py

```diff
@@ -182,8 +180,10 @@ def recur(syntax, lhs, rhs, var_idx, type_name):
             return recur_value(syntax, lhs, rhs, type_label)
 
         if encoding == 'mapping':
-            key_type_name = type_dict['key']
-            value_type_name = type_dict['value']
+            # TODO: Make less hacky once this is addressed: https://github.com/foundry-rs/foundry/issues/2462
+            type_contents = '('.join(type_name.split('(')[1:])[0:-1]
+            key_type_name, *rest = type_contents.split(',')
+            value_type_name = ','.join(rest)
             return recur_mapping(syntax, lhs, rhs, var_idx, key_type_name, value_type_name)
```

Comment:
This is to accomodate both `solc` output (which does already compute this for us as `key` and `value`), and `forge build` output, which discards this information for some reason.